src/scanners/css.js

```diff
@@ -12,7 +12,7 @@ export default class CSSScanner extends BaseScanner {
 
   _defaultRules = rules;
 
-  processCode(cssCode, cssInstruction) {
+  processCode(cssCode, cssInstruction, rules) {
```

Comment:
It is; good call. (I thought that myself but I think I was being lazy; no excuse!)